src/contour/TerminalSession.cpp

```diff
@@ -375,7 +374,15 @@ void TerminalSession::pasteFromClipboard(unsigned count)
         SessionLog()("pasteFromClipboard: mime data contains {} formats.", md->formats().size());
         for (int i = 0; i < md->formats().size(); ++i)
             SessionLog()("pasteFromClipboard[{}]: {}\n", i, md->formats().at(i).toStdString());
+#if !defined(_WIN32)
+        string const text = clipboard->text(QClipboard::Clipboard)
+                                .replace("\r\n", "\n")
+                                .replace("\r", "\n")
+                                .toUtf8()
+                                .toStdString();
+#else
```

Comment:
This logic is generally the same as below. Maybe we can put it into a private (anonymous namespace) helper function on the top part of this file, to avoid code duplication?